question27/js/news.js

```diff
@@ -47,6 +48,23 @@ init();
 function renderNewsUiElement(newsData) {
   tabsGroup.appendChild(getTabListsFragment(newsData));
   newsWrapper.appendChild(renderContents(newsData));
+  renderLogoutBtn();
+  logoutButtonEvent();
+}
+
+const renderLogoutBtn = () => {
+  const logoutButton = createElementWithClassName('button', 'logoutButton'); 
+  logoutButton.id = 'js-logout-btn';
+  logoutButton.textContent = 'Logout';
+  logoutButtonArea.appendChild(logoutButton);
+}
+
+const logoutButtonEvent = () => {
```

Comment:
IMO
I think the function name should contain a verb.

Reply:
Thank you for your review! You are right! I fixed the function name but what do you think? Please confirm.

fixed commit : https://github.com/csakyo/homework/pull/41/commits/6464546d7371a1b73aac51c0af15d8702bb1c525

Reply:
Thank you for your correction.

I think that the meaning is conveyed, so I think it's good,
I thought something like `setEventForLogoutButton` or `addEventForLogoutButton` would be more appropriate.
It may be a matter of taste, so I will approved.